content/guides/depin/getting-started.md

```diff
@@ -0,0 +1,208 @@
+
+See
+[example code](https://gist.github.com/lanvidr/88a594da06ba867bf8201fe8c6331dc0)
+and
+[Jupiter’s Merkle distributor](https://github.com/jup-ag/merkle-distributor-sdk)
+for an additional reference.
+
+#### Automatic Distribution
+
+An alternative to having users proactively claim rewards is automating rewards
+distribution. On a regular basis, this “rewards crank” fetches rewards for an
+associated set of users by querying and constructing transactions to distribute
+rewards to the specified accounts. Merkle tree updates can be posted by the
+automated process, allowing for the reward distribution mechanism to remain
+permissionless.
+
+### ZK Compression
+
+For networks that anticipate needing to distribute rewards to tens of thousands
+(or more) of nodes, participants, or contributors, a newer approach to rewards
+distribution is to use [ZK compression](https://www.zkcompression.com/). Instead
+of regular accounts, compressed accounts are generated for reward recipients,
+minimizing the state and rent costs associated with account creation.
+
+Implementing ZK compression is often cheaper in terms of storage costs. However,
+because it is a relatively new feature, the level of ecosystem support and
+tooling is not yet as extensive.
+
+See
+[example code](https://gist.github.com/lanvidr/4595f7b02236ffb2a3fb3ce9347ca044).
+
+### Cost analysis
+
+Let's estimate the cost of distributing rewards using both the Merkle tree
+approach and the ZK compression. We'll consider transaction fees, rent costs,
+and storage costs.
+
+In both approaches, updating the rewards per claim period requires a single
+transaction by the application, so the cost difference is minimal, as it doesn’t
+scale per the number of users. Both strategies require one transaction per user
+to claim their rewards.
+
+ZK compression is more cost-effective in storage costs, due to the reduced
+storage requirements of compressed data. Here is a hypothetical cost analysis to
+compare the storage costs.
+
+If we assume a reward distribution to 10,000 users, with an average reward
+amount per user of 100 tokens, and the transaction fee on Solana to be 0.000007
+SOL (7,000 lamports):
+
+Storage Costs using a Merkle tree distribution strategy:
+
+- The Merkle tree requires storing the leaf nodes and the internal nodes
+  - Leaf Nodes: 10,000 \* 32 bytes = 320,000 bytes
+  - Internal Nodes: (2^14 - 1) \* 32 bytes = 524,256 bytes
+  - Total Storage Cost: (320,000 + 524,256) \* 0.00000348 SOL/byte (per epoch) =
+    2.94 SOL
+- Total Cost (Merkle Tree): 0.050005 SOL + 0.00007323 SOL + 2.94 SOL = 2.99 SOL
+
+Storage Costs using a ZK compression distribution strategy:
+
+- Compressed Token Account: The compressed token account stores the compressed
+  reward data
+  - Compressed Data Size: Assuming a compression ratio of 50%, the total
+    compressed data size would be approximately 500 KB
+  - Total Storage Cost: 500 _ 1024 _ 0.00000348 SOL/byte (per epoch) = 1.78 SOL
```

Comment:
fixing the math symbols since they were converted to markdown italics:
```suggestion
- Total Storage Cost: 500 \* 1024 \* 0.00000348 SOL/byte (per epoch) = 1.78
    SOL
```